misc/flatpak/org.qutebrowser.qutebrowser.json

```diff
@@ -0,0 +1,197 @@
+                "make",
+                "make install"
+            ],
+            "ensure-writable": ["easy-install.pth"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://sourceforge.net/projects/pyqt/files/PyQt5/PyQt-5.9.2/PyQt5_gpl-5.9.2.tar.gz",
+                    "sha256": "c190dac598c97b0113ca5e7a37c71c623f02d1d713088addfacac4acfa4b8394"
+                }
+            ],
+            "modules": [
+                {
+                    "name": "sip",
+                    "buildsystem": "simple",
+                    "build-commands": [
+                        "python3 configure.py -d /app/lib/python3.5/site-packages -b /app/bin -e /app/include/python3.5 -v /app/share/sip",
+                        "make",
+                        "make install"
+                    ],
+                    "ensure-writable": ["easy-install.pth"],
+                    "sources": [
+                        {
+                            "type": "archive",
+                            "url": "https://sourceforge.net/projects/pyqt/files/sip/sip-4.19.6/sip-4.19.6.tar.gz",
+                            "sha256": "9dda27ae181bea782ebc8768d29f22f85ab6e5128ee3ab21f491febad707925a"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "flatpak-xdg-open-shim",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "git://github.com/mattdangerw/flatpak-xdg-open-shim",
+                    "commit": "7de8f1273c414f7d797d8487511861a7330b61fe"
+                }
+            ]
+        },
+        {
+            "name": "qutebrowser",
+            "buildsystem": "simple",
+            "build-commands": [
+                "python3 setup.py install --prefix=/app"
+            ],
+            "post-install": [
+                /* We currently only need the desktop, appstream and icon files */
+                "install -Dm644 misc/qutebrowser.appdata.xml /app/share/appdata/qutebrowser.appdata.xml",
+                "install -Dm644 misc/qutebrowser.desktop /app/share/applications/qutebrowser.desktop",
+                "install -Dm644 icons/qutebrowser.svg /app/share/icons/hicolor/scalable/apps/qutebrowser.svg",
+                "for SIZE in 16 24 32 48 64 128 256 512; do install -Dm644 icons/qutebrowser-${SIZE}x${SIZE}.png /app/share/icons/hicolor/${SIZE}x${SIZE}/apps/qutebrowser.png; done"
+            ],
+            "ensure-writable": ["easy-install.pth"],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/qutebrowser/qutebrowser.git",
+                    "tag": "v1.1.1",
```

Comment:
I suppose there isn't an easy way to re-use the information from e.g. `qutebrowser.appdata.xml` here?

Reply:
Flatpak requires reproducabilty of the builds. So it is kind of picky. If you use git you habe to specify the tag and the commit hash. If you use an archive you habe to speciy the hash sum of the file.

Reply:
Hmm, I see. Actually, using the release archive would probably be a better idea. Right now, `:help` gives you a 404 in the Flatpak as you don't generate the documentation (which is included in the releases).